package.json

```diff
@@ -8,7 +8,9 @@
     "clean": "rimraf ./app/node_modules",
     "format": "prettier --write '{src,config,test,locales}/**/*.{js,json}'",
     "precommit": "lint-staged",
-    "test": "cross-env NODE_ENV=test ava && npm run lint",
+    "test": "npm run test:unit && npm run test:gui && npm run lint",
+    "test:gui": "npm run build && cross-env NODE_ENV=test ava test/gui/**",
```

Comment:
This is slow... but the GUI test really runs against the built app, so without running `build` it isn't really testing the current code (plus the 1st time it did not work until you manually run build yourself)